demo/index.min.js

```diff
@@ -742,6 +752,7 @@ class AuroAccordion extends r {
           id="accordionTrigger"
           aria-controls="accordionContent"
           aria-expanded="${this.expanded}"
+          aria-disabled="${this.disabled ? 'true' : 'false'}"
```

Comment:
```suggestion
          ?aria-disabled="${this.disabled}"
```

You can just conditionally add the attribute based on `this.disabled`.

It's out of scope of this bug fix but `aria-expanded` should also actually be done the same way. Feel free to make that change too if you don't mind.

Reply:
sure, I will add this with additional commit message then.

Reply:
oh wait, but if we do this, then the aria-* attributes will just basically not available when the value is false.
Is that fine for the screen reader?

Also, we need to update every unit test that checks the attributes, from:
`hasAttribute('aria-expanded')).to.equal('false');` -> `hasAttribute('aria-expanded')).to.null;`
and
`hasAttribute('aria-expanded')).to.equal('true');` -> `hasAttribute('aria-expanded')).to.not.null;`